### PR TITLE
DBAPI: Add param to control capturing of db.statement.parameters

### PR DESCRIPTION
**Pull request by stschenk**
# Description

Switched DBAPI so that it will not capture parameterized query parameters.  Capturing this would be a problem in production environments since they will contain sensitive information such as session tokens and hashed passwords.

Have added a parameter to the instrumentation constructor to allow for this feature to be turned on.

I will mark this as a breaking change since I am removing the db.statement.parameter span attribute and something might be depending on it. 

Fixes https://github.com/open-telemetry/opentelemetry-python-contrib/issues/157

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit Tests

# Checklist:

- [ x] Followed the style guidelines of this project
- [ x] Changelogs have been updated
- [ x] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+Stop capturing query parameters by default
+  ([#156](https://github.com/open-telemetry/opentelemetry-python/pull/156))
```
Comment (toumorokoshi) [inline instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md:6]: should this be updated to reference this PR now?

File: instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py [hunk 2]
```diff
@@ -98,6 +101,7 @@ def wrap_connect(
     connection_attributes: typing.Dict = None,
     version: str = "",
     tracer_provider: typing.Optional[TracerProvider] = None,
+    capture_parameters: bool = False,
```
Comment (toumorokoshi) [inline instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py:104]: nitpick: with boolean values, I would suggest an auxiliary verb to indicate it's a boolean decision. 

e.g. "should_capture_parameters" .

File: instrumentation/opentelemetry-instrumentation-dbapi/tests/test_dbapi_integration.py [hunk 3]
```diff
@@ -53,6 +53,49 @@ def test_span_succeeded(self):
...
+        self.assertEqual(span.attributes["db.statement"], "Test query")
+        self.assertFalse("db.statement.parameters" in span.attributes)
+        self.assertEqual(span.attributes["db.user"], "testuser")
+        self.assertEqual(span.attributes["net.peer.name"], "testhost")
+        self.assertEqual(span.attributes["net.peer.port"], 123)
+        self.assertIs(
+            span.status.status_code, trace_api.status.StatusCode.UNSET,
+        )
+
+    def test_span_succeeded_without_capture_of_statement_parameters(self):
```
Comment (toumorokoshi) [inline instrumentation/opentelemetry-instrumentation-dbapi/tests/test_dbapi_integration.py:68]:
```suggestion
    def test_span_succeeded_with_capture_of_statement_parameters(self):
```

**Review by toumorokoshi — APPROVED** [review]
A couple small changes, but LGTM!

File: instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md (hunk 1 above)
Comment (stschenk) [reply instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md:6]: for sure, thanks!

File: instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py (hunk 2 above)
Comment (stschenk) [reply instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py:104]: AsyncPGInstrumentor already introduce the parameter as capture_parameters.  I am trying to be consistent with that.

**Review by lzchen — APPROVED** [review]

File: instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md [hunk 4]
```diff
@@ -16,7 +19,7 @@ Released 2020-09-17
 Released 2020-08-14
 
 - Change package name to opentelemetry-instrumentation-dbapi
-  ([#966](https://github.com/open-telemetry/opentelemetry-python/pull/966))
+  ([#156](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/156))
```
Comment (lzchen) [inline instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md:22]: Don't think we need this change.

Comment (stschenk) [reply instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md:22]: ugh...  not sure what I was thinking.  Will fix this.

**Pull request merged**